lib/common-helper.ts

```diff
@@ -89,7 +89,7 @@ export class CommonHelper {
     }
 
     escapeNameValue(value: string): string {
-        return value.replace(`'`, `\\'`);
+        return value.replace(/'/g, `\\'`);
```

Comment:
nitpick: you could use replaceAll

Please rename your commit according  convention - this link is in the comment in `Conversations` tab